src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -645,15 +675,17 @@ def _runtool_nonhermetic(self, parent_workunit, classpath, main, tool_name, args
       raise Exception('couldnt find work unit for underlying execution')
     return runjava_workunit
 
-  def _runtool(self, main, tool_name, args, distribution,
-    tgt=None, input_files=tuple(), input_digest=None, output_dir=None):
+  def _runtool(self, args, distribution,
+               tgt=None, input_files=tuple(), input_digest=None, output_dir=None):
+    main = 'rsc.cli.Main',
+    tool_name = 'rsc'
```

Comment:
This is a mildly orthogonal change, but I personally think it would make sense to keep this and also rename this method to `_invoke_rsc()` or something, since this method will always run rsc now (this was different before the metacp changes). This change could be entirely reverted, but I think the clarity provided by making it clear that only rsc is being invoked is a good reason to do it in this change.

Reply:
Should be fine.